Makefile

```diff
@@ -8,10 +8,10 @@ build:
 
 test: # @HELP run the unit tests and source code validation
 test: deps build lint license_check
```

Comment:
really 'build' is not required for test - the tests will run on their own if the build had not been run. I don't know how that compares to "go" Makefile structure